app/src/main/java/com/edu/movie/screen/base/HomePageFragment.kt

```diff
@@ -28,18 +32,22 @@ class HomePageFragment private constructor() : Fragment() {
             when (it.itemId) {
                 R.id.homePage -> {
                     textHeader.text = getText(R.string.movie)
+                    replaceFragment(HomeContentFragment.newInstance(), R.id.homePageContainer)
```

Comment:
Tạo viewpager adapter
https://stackoverflow.com/questions/58128491/how-to-combine-bottomnavigationview-and-viewpager